templates/sim_im_calc.sl.template

```diff
@@ -20,7 +22,7 @@ if [[ ! -d {{sim_dir}}/IM_calc ]]; then
 fi
 
 FD=$(getFromYaml "{{sim_dir}}/sim_params.yaml" FD_STATLIST)
-python $gmsim/workflow/scripts/im_calc_checkpoint.py {{sim_dir}}/IM_calc/ `wc -l ${FD}` --simulated
+python $gmsim/workflow/scripts/im_calc_checkpoint.py {{sim_dir}}/IM_calc/ `wc -l < ${FD}` --simulated
```

Comment:
Whats happening here? Am I missing something?

Reply:
If you pass the file name as an argument it will print it after the length of the file. If you pipe in the files contents it only shows the length of the file.

Reply:
![image](https://user-images.githubusercontent.com/46693562/54647108-3c981e80-4b06-11e9-8409-a53abf468c4a.png)

Reply:
Ahhhhhhhhhhhhh, quality bash magic haha